boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dts

```diff
@@ -86,6 +86,12 @@
 		label = "dac0";
 		#io-channel-cells = <1>;
 	};
+
```

Comment:
This needs to be moved out of this commit and added as a second one in this PR. CI still works OK in this case.

Reply:
Updated.